_includes/manuals/nightly/3.3.1_rpm_packages.md

```diff
@@ -55,12 +23,11 @@ Under each repo are directories for each distribution, then each architecture.
 
 ### Release packages
 
-To set up the repository, foreman-release packages are provided which add a repo definition to `/etc/yum.repos.d`.  Install the appropriate release RPM from these lists:
+To set up the repository, foreman-release packages are provided which add a repo definition to `/etc/yum.repos.d`. Install the release RPM and enable the module:
 
-    # EL7
-    yum localinstall https://yum.theforeman.org/releases/{{page.version}}/el7/x86_64/foreman-release.rpm
     # EL8
-    yum localinstall https://yum.theforeman.org/releases/{{page.version}}/el8/x86_64/foreman-release.rpm
+    dnf localinstall https://yum.theforeman.org/releases/{{page.version}}/el8/x86_64/foreman-release.rpm
```

Comment:
`localinstall` is deprecated in EL8 and now an alias for `install`. From `man dnf`:

```
   Install Command
       Command: install
       Aliases: in
       Aliases for explicit NEVRA matching: install-n, install-na, install-nevra
       Deprecated aliases: localinstall
```

```suggestion
    dnf install https://yum.theforeman.org/releases/{{page.version}}/el8/x86_64/foreman-release.rpm
```